zerver/tests/test_events.py

```diff
@@ -133,6 +133,14 @@ def test_events_register_endpoint(self):
             result = self.client_post('/json/register', **self.api_auth(email))
         self.assert_json_error(result, "Could not allocate event queue")
 
+        # Test that call is made to deal with a returning soft deactivated user.
+        with mock.patch('zerver.lib.events.maybe_catch_up_soft_deactivated_user', return_value='15:11') as fa, \
```

Comment:
The return_value looks like a copy/paste error.  If a function's return value is None or is not used by the caller, then I would not specify anything in the mock.